.gitignore

```diff
@@ -7,4 +7,5 @@ docs/src/jekyll/*.md
 docs/target/*
 combinators/target/*
 magic/target/*
-datetime/target/*
\ No newline at end of file
+datetime/target/*
+.idea/*
```

Comment:
This should be removed, should go in a personal gitignore rather than the project one. See https://stackoverflow.com/questions/6117109/few-questions-with-git-possible-to-have-custom-gitignore-read-only-access